### ISSUE TITLE
[Major Feature Request] Plugin Manager

### ISSUE THREAD

**Issue by paladini** [koreader/koreader#1711]
Hello guys!

I have a Kobo Aura (not HD) where I run Koreader instead of the default Kobo reader. **I would like to suggest an interesting feature: a Koreader installer.** 

_Please, comment only after read this post completely._

<hr>
## How

The idea is to provide a Koreader installer in the first time you run Koreader. In the installation process, you should be able to configure which plugins you do like to install. This includes every "core plugin" we have now on Koreader default version:
- Dictionary support
- Wikipedia support
- Evernote export
- Progress Sync
- Zsync transport
- Statistics
- Etc.

When you choose to not install a "core plugin", this plugin will not appear as an option in the Koreader menu. In other words, the Koreader screens will be simpler, displaying only the plugins you really want to use. If I don't want to install Statistics plugin, you don't need to and this menu option will not appear for you, freeing your (almost small) screen from unnecessary options.

The "core plugins" still can be installed and uninstalled from a new Koreader option in the menu. There you can check which is the enabled and disabled plugins.
## Why

This is a nice feature if you want to make Koreader scalable. For example: in the future you can create a "Plugin Central" for Koreader, where people can submit their own plugins without the need to have a Pull Request accepted in the Github and the code incorporated to the official and default Koreader client. 

The other main benefit from a feature like that is that you can provide a highly customizable software for the users. Everyone loves that, who knows what end users can do with some freedom :3 

A smaller benefit would be (possibly) the power save, because there's no need to execute the instructions of unused plugins (even if it was just a set of conditions).

I really would love a feature like that and I think a lot of people would love it too. Just think in how many plugins can be created by the users and how this will improve the Koreader community and Koreader features. 

**What do you think about this proposal?**

**Comment by chrox** [comment]
I would call this "Plugin Manager" and we will definitely have this after we have 5 another plugins. But now it should be of low priority.

**Comment by paladini** [comment]
I've changed the name of the issue, thanks for the feedback.

**Comment by NiLuJe** [comment]
c.f., #2544
